cmd/kubeless/list_test.go

```diff
@@ -41,8 +43,23 @@ import (
 )
 
 func fakeCRDClient(f func(req *http.Request) (*http.Response, error)) *restFake.RESTClient {
+	reg := api.Registry
+	legacySchema := schema.GroupVersion{
+		Group:   "",
+		Version: "v1",
+	}
+	crdSchema := schema.GroupVersion{
+		Group:   "k8s.io",
+		Version: "v1",
+	}
+	reg.RegisterGroup(apimachinery.GroupMeta{
```

Comment:
These are registering the new GroupVersion into the *global* `api.Registry`.  Either auto-register into the global from an `init()` function somewhere, or use a registry isolated to this fake CRD client (see `apimachinery/pkg/apimachinery/registered.NewOrDie()`) - otherwise using this test function will pollute the global environment and lead to unrepresentative results.